src/content/docs/faq.md

```diff
@@ -347,6 +347,25 @@ lines. Here's an example using [`tui-big-text`](https://github.com/joshka/tui-bi
 
 ![[tui-big-text](https://github.com/joshka/tui-big-text/)](https://camo.githubusercontent.com/3a738ce21da3ae67660181538ef27473b86bebca73f42944e8012d52f86e500d/68747470733a2f2f7668732e636861726d2e73682f7668732d3364545474724c6b79553534684e61683232504152392e676966)
 
+## Some characters appear to be missing / look weird
+
+`ratatui`, and TUIs in general, use special drawing characters like [box-drawing characters],
+[braille characters] or even icons. If your font doesn't support such features, it may display a
+white square (□) or a replacement character (�).
+
+[box-drawing characters]: https://en.wikipedia.org/wiki/Box-drawing_character
+[braille characters]: https://en.wikipedia.org/wiki/Braille_Patterns#Block
+
+To fix this, we recommend you use a [nerd fonts], this is generally what works the best. Though note
+that some characters may render a bit differently from font to font.
```

Comment:
Do we want to recommend a particular font? I know I had some "problems" with JetBrains nerdfont here https://github.com/ratatui-org/ratatui/pull/831#issuecomment-1893604623 so this one it out.

Reply:
https://github.com/be5invis/Iosevka has the best all round symbol support, but you need to use the extended version to be a normal width (the regular width is what other fonts might call narrow). 

I personally like and use FiraCode Nerd Font Mono as it seems to have a good balance compared to all the other fonts I've tried. "Hack" was runner up, with "Meslo LG" a close third.

There's also http://www.kreativekorp.com/software/fonts/ksquare/ which is potentially good as a fallback font if needed.

> I know I had some "problems" with JetBrains nerdfont here https://github.com/ratatui-org/ratatui/pull/831#issuecomment-1893604623 so this one it out.

I think most fonts get the widths of the eighth and quarter blocks wrong, and most fonts don't extend those characters to outside the box so that there isn't a gap when rendering.  Iosevka mentioned above is the closest I've found at getting some of that right.

Reply:
I've added a mention to ksquare only because ultimately every font is different and the others you mentioned are all on nerdfonts